rotkehlchen/chain/polygon_pos/constants.py

```diff
@@ -5,6 +5,7 @@
 
 POLYGON_POS_ETHERSCAN_NODE_NAME = 'polygon pos etherscan'
 POLYGON_POS_GENESIS = Timestamp(1590824836)
+POLYGON_POS_POL_HARDFORK = Timestamp(1725408000)  # TODO: Add a more precise timestamp
```

Comment:
According to David from Polygon it's https://x.com/davidesilverman/status/1828445312618320150

```suggestion
POLYGON_POS_POL_HARDFORK = Timestamp(1725451200)
```